Mono.Addins.Setup/Mono.Addins.Setup/AddinStore.cs

```diff
@@ -49,7 +49,7 @@
 
 namespace Mono.Addins.Setup
 {
-	internal class AddinStore
+	public class AddinStore
```

Comment:
Don't change. If you need DownloadFile, move it to another class that is already public.